.env

```diff
@@ -157,8 +157,7 @@ VALKEY_ADDR=valkey-cart:${VALKEY_PORT}
 # Postgres
 POSTGRES_HOST=postgresql
 POSTGRES_PORT=5432
-POSTGRES_DB=otel
```

Comment:
keep the default database instance `postgres` and create a dedicated DB instance `shop_db` for the application, see `init.sql` for the creation of this `shop_db` instance.